### PR TITLE
15 feature login page

### PR DESCRIPTION
**Pull request by ElaM8**
Adding login page, minus federated sign in component, including authentication error message and testing. Amended README to reflect changes.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -60,13 +60,6 @@ const fakeStyle = {
     ```
 ````
 
-## SplashPage
```
Comment (Charlie-robin) [inline README.md:63]: Do you need to delete this from the readme?

File: src/pages/Login/Login.test.tsx [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+import userEvent from "@testing-library/user-event"
+
+  const email = "daniela.gutperl@gmail.com"
+  const setEmail = () => {return}
+  const password = "password"
+  const setPassword = () => {return}
+  const setUserId = () => {return}
+
+it("should render elements to allow login", () => {
+  render(<MemoryRouter><Login
```
Comment (Charlie-robin) [inline src/pages/Login/Login.test.tsx:13]: You can replace this `render` with `customRender` from `src/utils/testUtils.tsx`.

File: src/pages/Login/Login.tsx [hunk 3]
```diff
@@ -13,35 +17,60 @@ type LoginProps = {
...
-  const navigate = useNavigate();
+  const navigate = useNavigate()
+  const [loginError, setLoginError] = useState<boolean>(false);
 
   const handleLogin = async () => {
     try {
       const userData = await signInWithEmailAndPassword(auth, email, password);
       setUserId(userData.user.uid);
-      navigate("/home");
+      navigate("/home")
```
Comment (Charlie-robin) [inline src/pages/Login/Login.tsx:35]: I do not think you want to navigate to "/home" here as that is not the route for `Home` anymore

**Review by Charlie-robin — CHANGES_REQUESTED** [review]

**Comment by Charlie-robin** [comment]
Hey, some tweaks to make in the comments above can you also resolve the merge conflicts?

**Review by Charlie-robin — APPROVED** [review]
Looks good to me 👍

**Review by olirob93 — APPROVED** [review]
all good

**Pull request merged**